src/commands/local-store/local-store-command.js

```diff
@@ -93,8 +93,11 @@ class LocalStoreCommand extends Command {
                 await this.commandExecutor.add({
                     name: 'deletePendingStateCommand',
                     sequence: [],
-                    delay: updateCommitWindowDuration * 1000,
-                    data: { ...command.data, repository: PENDING_STORAGE_REPOSITORIES.PRIVATE },
+                    delay: (updateCommitWindowDuration + 60) * 1000,
```

Comment:
Why have we introduced here this additional 60 seconds delay? Maybe we can introduce new constant for this, or at least leave a comment for it later